documentation/TagAnalysis.md

```diff
@@ -0,0 +1,206 @@
+# Tag Analysis
+
+## Overview
+
+Tag analysis is a compile-time static analysis that keeps track of information flows in programs. Information flows
+from value X to value Y, whenever information stored in X is transferred to value Y. Tag analysis attaches different
```

Comment:
When we add a tag to a variable, the tag is attached to the value bound by the variable. I think "object" could be a proper name, but that might be confusing because Rust is not object-oriented.

Reply:
The terms are a bit loose. In this context there is no difference between information and values. If you can store something in X, then X is a place/location/variable, not a value or information, the latter being the thing that you store in the place/location/variable.